fabtests/functional/msg_sockets.c

```diff
@@ -263,7 +263,9 @@ static int client_connect(void)
 
 	/* Close the passive endpoint that we "stole" the source address
 	 * from */
-	FT_CLOSE_FID(pep);
+	ret = ft_close_fid(pep);
+	if (ret)
+		return ret;
```

Comment:
We get a similar crash in msg_sockets attempting to close the pep twice, once here and again with ft_free_res() at the end of the test.